app/controllers/users_controller.rb

```diff
@@ -6,4 +6,39 @@ def index
   def show
     @user = User.find(params[:id])
   end
+
+  def new
+    @user = User.new
+  end
+
```

Comment:
- [ ] After adding two users and running your app, it turns out their not been added to the database as `ids` were not even assigned to them. Kindly review this and make sure added users are been added

### View:
![image](https://github.com/microhass/blog-app/assets/50990925/1d82d527-6498-485c-8a7e-2c65a4bab8f7)